topologies/kne/init_arista_gribi.config

```diff
@@ -0,0 +1,44 @@
+transceiver qsfp default-mode 4x10G
```

Comment:
Can we just fold the gribi config into the one arista config?

Reply:
Yes, sure